testgrid/config.yaml

```diff
@@ -2617,10 +2617,18 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gce-reboot
     description: "Uses kubetest to run a subset of e2e tests (+Feature:Reboot) against a cluster created with cluster/kube-up.sh"
 
+- name: sig-release-1.15-blocking
+- name: sig-release-1.15-informing
+- name: sig-release-1.15-all
```

Comment:
We are not going to be having `-all` for 1.15 I believe. We are transitioning to `-informing`. As far as I remember we decided to do this incrementally (fork from master-blocking and master-informing into the new releases and ultimately `-all` dashbaords will retire once 1.14 release expires).

Reply:
We could remove `-all` dashboard in the coming weeks incrementally.

Reply:
ref: https://github.com/kubernetes/test-infra/issues/11977#issuecomment-492450273 for dropping -all